### DISCUSSION TITLE
Adding country

### DISCUSSION THREAD

**Discussion by mochouinard** [Daniel-Ioannou/flutter_country_picker#119] [Ideas]
Hi,

I would be interested into adding a custom country.

Basically, a lot of people in the province of Québec in Canada don't feel like they are part of the Country Canada.
So I would like to have a default initial country of Québec, and have Québec in the list as well, which would have the same data as canada except the name and the flag icon.

Anyway to do it without modifying the base code ?

Thanks

PS: I'm not to dart and flutter as a few hours ago.

**Comment by Daniel-Ioannou** [comment]
Hi, 
Unfortunately, currently there is no way to add a custom country. 
You can find all the countries used by this package [HERE](https://github.com/Daniel-Ioannou/flutter_country_picker/blob/master/lib/src/res/country_codes.dart).
